code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -3302,8 +3302,178 @@
 	description = "An alcoholic beverage made from lightly fermented Sareszhi berries, considered an upper class delicacy on Moghes. Significant butanol content indicates intoxicating effects on Unathi."
 	color = "#bf8fbc"
 	strength = 20
-	taste_description = "grape soda"
+	taste_description = "berry juice"
 
 	glass_icon_state = "sarezhiglass"
 	glass_name = "glass of Sarezhi Wine"
-	glass_desc = "It tastes like flat grape soda. Is this supposed to be alcoholic?"
+	glass_desc = "It tastes like plain berry juice. Is this supposed to be alcoholic?"
+
+//Kaed's Unathi Cocktails
```

Comment:
Generally putting names in source is discouraged.

Reply:
Sorry, all the other new drink subsections are labelled by people's names "/Brightdawns super cool coffee drinks// etc, so I thought I was supposed to.  Do you want me to remove it?

Reply:
It's a Brightdawn-ism, understandable if you thought it'd be fine. Usually not needed to put code comments unless it's explaining a code snippet or something that shouldn't be removed/touched.